### PR TITLE
added subnet to function app within empinc

### PR DESCRIPTION
**Pull request by cpwevans**

### CODE REVIEW

File: azure/template.json [hunk 1]
```diff
@@ -99,6 +99,18 @@
         },
         "nServiceBusLicense": {
             "type": "string"
+        },
+        "sharedEnvVirtualNetworkName": {
+            "type": "string"
+        },
+        "subnetObject": {
+            "type": "string"
```
Comment (adam230594) [inline azure/template.json:107]: Is this correct?

File: azure/template.json [hunk 2]
```diff
@@ -99,6 +99,18 @@
...
+        },
+        "subnetServiceEndpointList": {
+            "type": "string"
```
Comment (adam230594) [inline azure/template.json:110]: Is this correct?

File: azure/template.json [hunk 3]
```diff
@@ -99,6 +99,18 @@
...
+        },
+        "subnetDelegations": {
+            "type": "string"
```
Comment (adam230594) [inline azure/template.json:113]: Is this correct?

File: azure/template.json (hunk 1 above)
Comment (cpwevans) [reply azure/template.json]: Updated it 👍

File: azure/template.json (hunk 2 above)
Comment (cpwevans) [reply azure/template.json]: Updated it 👍

File: azure/template.json (hunk 3 above)
Comment (cpwevans) [reply azure/template.json]: Updated it 👍

**Review by jack-education — APPROVED** [review]

**Pull request merged**